contentcuration/contentcuration/frontend/channelList/views/Channel/CatalogFilters.vue

```diff
@@ -1,233 +1,75 @@
     >
-      <VContainer class="filters pa-3">
-        <VToolbar
-          v-if="$vuetify.breakpoint.xsOnly"
-          color="transparent"
-          flat
-          dense
-        >
-          <VSpacer />
-          <VBtn
-            icon
-            flat
-            style="text-align: right"
-            @click="drawer = false"
-          >
-            <Icon icon="clear" />
-          </VBtn>
-        </VToolbar>
-
-        <!-- Keyword search -->
-        <VTextField
-          v-model="keywordInput"
-          color="primary"
-          :label="$tr('searchLabel')"
-          box
-          clearable
-          data-test="keywords"
-          autofocus
-          @input="setKeywords"
-        />
-
-        <!-- Language -->
-        <LanguageFilter
-          v-model="languages"
-          :menu-props="menuProps"
-        />
-
-        <!-- License (attach to self to keep in notranslate class) -->
-        <MultiSelect
-          v-if="!libraryMode"
-          v-model="licenses"
-          :items="licenseOptions"
-          :label="$tr('licenseLabel')"
-        />
-
-        <!-- Formats (attach to self to keep in notranslate class) -->
-        <MultiSelect
-          v-model="kinds"
-          :items="kindOptions"
-          :label="$tr('formatLabel')"
-        />
-
-        <!-- Starred -->
-        <Checkbox
-          v-if="loggedIn"
-          v-model="bookmark"
-          :label="$tr('starredLabel')"
-        />
-
-        <!-- Includes -->
-        <div class="subheading">
-          {{ $tr('includesLabel') }}
-        </div>
-
-        <div :style="{ display: 'flex', alignItems: 'center' }">
-          <Checkbox
-            v-model="coach"
-            aria-describedby="tooltip-coach"
-            :label="$tr('coachLabel')"
-          />
-          <HelpTooltip
-            :text="$tr('coachDescription')"
-            maxWidth="250px"
-            tooltipId="tooltip-coach"
-          />
-        </div>
-
-        <Checkbox
-          v-model="subtitles"
-          :label="$tr('subtitlesLabel')"
-        />
-        <KRouterLink
-          class="qa-link"
-          :to="faqLink"
-          :text="$tr('frequentlyAskedQuestionsLink')"
-          appearance="basic-link"
-          iconAfter="openNewTab"
-          target="_blank"
-        />
-      </VContainer>
-      <VFooter
-        class="pb-3 pt-2 px-4"
-        color="transparent"
-        height="100"
-      >
-        <div>
-          <VImg
-            height="60"
-            width="90"
-            class="mb-1 mr-2"
-            contain
-            :src="require('shared/images/le-logo.svg')"
-          />
-          <KExternalLink
-            href="https://learningequality.org/"
-            :text="$tr('copyright', { year: new Date().getFullYear() })"
-            openInNewTab
-          />
-        </div>
-      </VFooter>
-    </VNavigationDrawer>
+      <CatalogFilterPanelContent />
+    </aside>
+    <SidePanelModal
+      v-if="windowIsSmall && showSidePanel"
```

Comment:
Could we be more specific with this `showSidePanel` variable? 😅. I think something like `showCatalogFiltersSidePanel` would be great :)